types/rangy/lib/rangy-classapplier.d.ts

```diff
@@ -0,0 +1,120 @@
+// tslint:disable-next-line:no-self-import
```

Comment:
Do not want these rules here.

Reply:
this rule must be disable because:
+ The consumer code need `import 'rangy/lib/rangy-classapplier'` to use rangy-classapplier.
+ In `rangy-classapplier.d.ts`, if I `import '../index' then I cannot augment module rangy (must enable rule `no-declare-current-package`)
+ But if no-declare-current-package, by copy content of `rangy-classapplier.d.ts` into `index.d.ts` then the consumer code can use rangy-classapplier directly without `import 'rangy/lib/rangy-classapplier'` - which cause error at run time!